rapstore-frontend/src/environments/environment.dev.ts

```diff
@@ -1,4 +1,5 @@
 export const environment = {
-  production: false,
-  apiUrl: "http://api.dev.riot-apps.net"
+    production: false,
+    apiUrl: "http://api.dev.riot-apps.net",
+    appVersion: "0.1.0 BETA"
```

Comment:
check [this entry](https://stackoverflow.com/questions/34907682/how-to-display-app-version-in-angular2)